tests/test_rhel_compose.py

```diff
@@ -142,8 +142,9 @@ def mock_fetch_rpm_data(compose_url, variants):
 
 
 @pytest.mark.django_db
+@patch("corgi.tasks.brew.slow_fetch_modular_build.delay")
 @patch("corgi.collectors.rhel_compose.RhelCompose._fetch_rpm_data", new=mock_fetch_rpm_data)
-def test_save_compose(requests_mock):
+def test_save_compose(mock_fetch_modular_build, requests_mock):
```

Comment:
I just added the mock so that the task didn't fail because celery/redis is not setup for accepting delayed tasks. The focus of this test is on the save_compose part, not the create_relation it calls. I added a new test for that.